Node/angular-karma/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "angular-proj1",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "rimraf rimraf ./a11y-results && ng test"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular-builders/custom-webpack": "^14.1.0",
+    "@angular/animations": "^14.2.0",
+    "@angular/common": "^14.2.0",
+    "@angular/compiler": "^14.2.0",
+    "@angular/core": "^14.2.0",
+    "@angular/forms": "^14.2.0",
+    "@angular/platform-browser": "^14.2.0",
+    "@angular/platform-browser-dynamic": "^14.2.0",
+    "@angular/router": "^14.2.0",
+    "querystring-es3": "^0.2.1",
+    "rxjs": "~7.5.0",
+    "tslib": "^2.3.0",
+    "tty-browserify": "^0.0.1",
+    "url": "^0.11.0",
+    "util": "^0.12.5",
+    "zone.js": "~0.11.4"
+  },
+  "devDependencies": {
+    "@angular-devkit/build-angular": "^14.2.5",
+    "@angular/cli": "~14.2.5",
+    "@angular/compiler-cli": "^14.2.0",
+    "@axe-devtools/browser": "^4.6.1",
+    "@axe-devtools/karma-json-reporter": "^4.6.1",
+    "@axe-devtools/reporter": "^4.5.1",
+    "@types/jasmine": "~4.0.0",
+    "assert": "^2.0.0",
+    "bootstrap": "^5.2.2",
+    "install": "^0.13.0",
+    "jasmine-core": "~4.3.0",
+    "jquery": "^3.6.1",
+    "karma": "~6.4.0",
+    "karma-chrome-launcher": "~3.1.0",
+    "karma-coverage": "~2.2.0",
+    "karma-jasmine": "~5.1.0",
+    "karma-jasmine-html-reporter": "~2.0.0",
+    "node-polyfill-webpack-plugin": "^2.0.1",
+    "npm": "^9.6.5",
```

Comment:
Is this required?

Reply:
Karma-jasmin and karma-jasmine are required.

Do not need the other 2 i.e.
    "node-polyfill-webpack-plugin": "^2.0.1",
    "npm": "^9.6.5",
Hence removing them.